analysis/anx_dep_proms/proms_lists_anx_dep.qmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "proms_lists_anx_dep"
+format: html
+editor: visual
+---
+
+# Patient-Reported Outcome Measures (PROMs) Lists for Anxiety and Depression
+
+The following PROMs are required or recommended in NHS Talking Therapies for patients presenting with depression or anxiety, based on NICE guidelines and the NHS Talking Therapies Manual (Table 8). Additional PROMs identified through SNOMED codelists are also included.
```

Comment:
great to have all this information here, similar to the comment above, would be good to add the source here, so we know where you looked so far. the simplest solution would be just to add the hyperlink for the NHS Talking Therapies Manual here for now